com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -94,6 +94,55 @@ modules:
       - *proton_source
 
 
+  - name: widl
+    subdir: wine
+    build-options:
+      arch:
+        x86_64:
+          config-opts:
+            - --enable-win64
+          libdir: /app/share/steam/compatibilitytools.d/Proton-Exp/lib
+    make-args:
+      - tools/widl/install
+    no-make-install: true
```

Comment:
I think this deservers a bit of explanation. Why we're building widl separately? It's available in the mingw-w64 toolchain. Also I don't see how Wine's widl here is installed here.

Reply:
Oh I didn't know it was part of mingw-w64 toolchain.

widl is needed to build vkd3d from git source as it's used to generate some `.h` files and the build fails without it. It couldn't find the widl binary from mingw-w64, that's why I added this module.

As for the installation part, the make target `tools/widl/install` take care of compiling and installing widl.

Reply:
Should be better now.

Reply:
Does the upstream use mingw's or wine's widl?

Reply:
I would guess mingw's one since they don't build it in their Makefile behorehand.